data/mods/Aftershock/items/ammo/6.5mm.json

```
@@ -0,0 +1,44 @@
+[
+  {
+    "id": "afs_6.5mm_caseless",
+    "type": "AMMO",
+    "name": { "str_sp": "6.5mm caseless" },
+    "description": "An intermediate bullet, popularized during the first years of the Reclamation, these bullets are interchangeable between most modern assault rifles and machine guns.  These rounds being caseless cannot be disassembled or reloaded.",
+    "weight": "12 g",
+    "volume": "194 ml",
+    "price": 280,
+    "price_postapoc": 900,
+    "material": [ "brass", "powder" ],
+    "symbol": "=",
+    "color": "yellow",
+    "count": 30,
+    "stack_size": 47,
+    "ammo_type": "afs_6.5mm",
+    "range": 36,
+    "damage": { "damage_type": "bullet", "amount": 44, "armor_penetration": 2 },
+    "dispersion": 30,
+    "recoil": 1500,
+    "effects": [ "COOKOFF" ]
+  },
+  {
+    "id": "afs_6.5mm_lr",
+    "type": "AMMO",
+    "name": { "str_sp": "6.5mm LR" },
+    "description": "6.5mm Long Rifle ammunition with unjacketed bullets.  The 6.5mm LR round is extremely weak with very low stopping power, short range, and negligible recoil.  It is most extremely cheap to manufacture, and is only useful for rifle training, and hunting small animals.",
+    "weight": "3 g",
+    "volume": "65 ml",
+    "price": 150,
+    "price_postapoc": 800,
+    "material": [ "brass", "lead", "powder" ],
```

```suggestion
    "material": [  "lead" ],
```
Maybe we need to actually remove the brass above and replace them with lead and in heavy duty rounds DUP (depleted Uranium)

---

Agreed